workloads/templates/workload-fio-script-cm.yml.j2

```diff
@@ -0,0 +1,244 @@
+      name: fiotest-test
+    objects:
+    - apiVersion: v1
+      kind: PersistentVolumeClaim
+      metadata:
+        name: "${PVC_NAME}"
+        annotations:
+          volume.beta.kubernetes.io/storage-class: {{ fiotest_storageclass }}
+      spec:
+        accessModes:
+         - {{ access_modes }}
+        resources:
+          requests:
+            storage: {{ fiotest_storage_size }}
+    - apiVersion: v1
+      kind: ReplicationController
+      metadata:
+        labels:
+          name: fio
+        name: fiotest-pod-${IDENTIFIER}
+      spec:
+        replicas: 1
+        selector:
+          name: fio
+        template:
+          metadata:
+            labels:
+              name: fio
+            name: fio
+          spec:
+            hostNetwork: false
+            containers:
+              - name: "fio"
+                image: {{ fiotest_pod_image }}
+                imagePullPolicy: Always
+                env:
+                  - name: FIO_SSH_PORT
+                    value: "${FIO_SSHD_PORT}"
+                securityContext: {}
+                volumeMounts:
+                  - name: pbench-ssh
+                    mountPath: /.ssh/authorized_keys
+                    subPath: authorized_keys
+                  - name: pbench-ssh
+                    mountPath: /root/.ssh/authorized_keys
+                    subPath: authorized_keys
+                  - name: pbench-results
+                    mountPath: /var/lib/pbench-agent
+                  - name: persistentvolume
+                    mountPath: "/mnt/pvcmount"
+            dnsPolicy: ClusterFirst
+            securityContext: {}
+            restartPolicy: Always
+            volumes:
+              - name: pbench-ssh
+                secret:
+                  secretName: pbench-ssh
+                  defaultMode: 0600
+              - name: pbench-results
+                emptyDir: {}
+              - name: persistentvolume
+                persistentVolumeClaim:
+                  claimName: "${PVC_NAME}"
+    parameters:
+    - name: PVC_NAME
+      description: PVC name
+      required: true
+      from: pvc[a-z0-9]{10}
+      generate: expression
+    - name: IDENTIFIER
+      description: Number to append to the name of resources
+      value: '1'
+    - name: FIO_SSHD_PORT
+      description: What port should the sshd process in the fio container listen on
+      value: "2022"
+      required: true
+    - name: STORAGE_FIO_IMAGE
+      description: Which fio container image to be used in this pod
+      value: "quay.io/openshift-scale/scale-ci-fio"
+      required: true
+  pbench-ssh.yaml: |
+    kind: Template
+    apiVersion: v1
+    metadata:
+      name: sshSecretTemplate
+      creationTimestamp:
+      annotations:
+        description: Template to create ssh secret
+        tags: ''
+    objects:
+      - apiVersion: v1
+        kind: Secret
+        metadata:
+          name: pbench-ssh
+          namespace: {{ fiotest_basename }}0
```

Comment:
I am not sure the namespace is needed here as cluster loader should place this in the correct project/namespace.

Reply:
agree, this can be removed